pythonwhat/tasks.py

```diff
@@ -447,7 +447,7 @@ def __init__(self, **kwargs):
     def __call__(self, shell):
         try:
             with capture_output() as out:
-                get_env(shell.user_ns)[self.fun_name](*self.arguments)
+                get_env(shell.user_ns)[self.fun_name](*self.arguments['args'], **self.arguments['kwargs'])
```

Comment:
Not 100% sure where these arguments come from, but all arguments are used here, right? Not just `*args` and `**kwargs`?